### ISSUE TITLE
Avoid classes with only static members

### ISSUE THREAD

**Issue by matanlurey** [dart-lang/linter#310]
They probably mean to define a namespace, but the `as` prefix and top-level members are preferred in Dart, i.e. this is bad:

``` dart
class StringUtils {
  static String capitalize(String s) => ...
}
```

**Comment by ksikka** [comment]
Say you want to pass an object as an export to an Angular template that contains all the messages.
You can't pass the as prefix, so you have to create a class with static members.

If you know of an alternative way to wrap messages in a single object so they can be neatly passed to a template, I'm interested.

The reason I'm trying to wrap the messages in an object separate from the component is so that the messages can be used in multiple components.

**Comment by a14n** [comment]
Fixed some times ago by #540 (lint `avoid_classes_with_only_static_members`)
